src/extractor/scripting_environment.cpp

```diff
@@ -43,6 +44,11 @@ template <class T> double lonToDouble(T const &object)
     return static_cast<double>(util::toFloating(object.lon));
 }
 
+// Luabind does not like memr funs: instead of casting to the function's signature (mem fun ptr) we simply wrap it
```

Comment:
You are right, this was my try at making `osmium::Node`s available in the `way_function`.
Which is not possible, all we get are `osmium::NodeRef`s and they only have an id but no tags and accessing their location crashes since we're not using `libosmium`'s `NodeLocationsForWays` handler :sigh:. Splitting out the commit.